cmake/adjust/template/vendor/CXX_Clang.cmake

```diff
@@ -0,0 +1,51 @@
+include_guard(GLOBAL)
+
+# Debug
+set(NBL_CXX_DEBUG_COMPILE_OPTIONS
+	-ggdb3 -Wall -fno-omit-frame-pointer -fstack-protector-strong
+)
+
+# Release
+set(NBL_CXX_RELEASE_COMPILE_OPTIONS
+	-fexpensive-optimizations
+)
+
+# RelWithDebInfo
+set(NBL_CXX_RELWITHDEBINFO_COMPILE_OPTIONS "")
```

Comment:
what about O1 or something ? Alos don't you need `ggdb3` ?

Reply:
> what about O1 or something ? Alos don't you need `ggdb3` ?

https://github.com/Devsh-Graphics-Programming/Nabla/pull/791#discussion_r2033336186 + resolved